index.html

```diff
@@ -8,31 +8,33 @@
   <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet">
   <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/js/bootstrap.bundle.min.js"></script>
   <link rel="stylesheet" href="./Style.css" />
-  <link rel="stylesheet" href="./app.css">
+  <script defer src="./Script.js"></script>
   <title>Conference on Web Development</title>
-</head>
-<section id="navbar">
+  </head>
+  <body>
+  <section id="navbar">
   <nav class="navbar navbar-expand-lg navbar-light bg-light">
     <div class="container-fluid">
       <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarTogglerDemo03" aria-controls="navbarTogglerDemo03" aria-expanded="false" aria-label="Toggle navigation">
         <span class="navbar-toggler-icon"></span>
       </button>
-      <a class="navbar-brand" href="#"href="#index"><img src="./images/Logo.png"><a alt="Epal Logo"><a alt="Epal Logo"></a></a>
+      <a class="navbar-brand" href="#index"><img src="./images/Logo.png" alt="Epal Logo">EPALTechs</a>
       <div class="collapse navbar-collapse" id="navbarTogglerDemo03">
         <ul class="navbar-nav me-auto mb-2 mb-lg-0">
           <li class="nav-item">
-            <a class="nav-link" href="./about.html" class="link">About</a>
+          <a class="nav-link" href="./about.html" id="link">About</a>
           </li>
+          <li>
           <a class="nav-link active" aria-current="page" href="#">Program</a>
           </li>
-          <a class="nav-link active" aria-current="page" href="#">Contact</a>
+          <li><a class="nav-link active" aria-current="page" href="#">Contact</a>
           </li>
-          <a class="nav-link active" aria-current="page" href="#">Sponsor</a>
+          <li><a class="nav-link active" aria-current="page" href="#">Sponsor</a>
           </li>
-          <a class="nav-link active" aria-current="page" href="#">Join</a>
+          <li><a class="nav-link active" aria-current="page" href="#">Join</a>
           </li>
           <li class="nav-item">
-            <a class="nav-link" href="#">Link</a>
+            <a class="nav-link" href="mailto:epaltechs@gmail.com">Link</a>
```

Comment:
for the `navbar` please implement the following changes:

- [ ] Kindly add the dark top navbar in your desktop view and add social links in it to match the original design.
- [ ] Kindly add the correct width value to your nave bar and make it take the whole page width like the original design.
- [ ] Please reposition the nav links to the right and remove the search bar to match the original design and remove the unrelated nave links like `link` and `disabled`, and kindly next time when you use a ready bootstrap template edit it and remove all other boilerplate code that you don't need.

| Original Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/164032748-b060350d-22e3-4e81-92f9-7aae2cc1c681.png)  | ![image](https://user-images.githubusercontent.com/68796232/164032776-c7067b9a-e633-42bf-bb5c-b33fe40a84c2.png)  |